src/api/c/ops.hpp

```diff
@@ -25,82 +25,51 @@ using namespace detail;
 
 template<typename T, af_op_t op>
 struct Binary {
-    static __DH__ detail::compute_t<T> init();
```

Comment:
We moved to this `compute` change not too long ago. What change resulted in this inversion ?